centreon/www/main.php

```diff
@@ -1,6 +1,6 @@
 <?php
 /*
- * Copyright 2005-2015 Centreon
+ * Copyright 2005-2023 Centreon
  * Centreon is developped by : Julien Mathis and Romain Le Merlus under
```

Comment:
```suggestion
 * Centreon is developed by : Julien Mathis and Romain Le Merlus under
```

Reply:
I remove this change, it was only for triggering purpose xD